packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1273,6 +1296,15 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       this.setState({ startingString: '' })
     }, Dropdown.charKeyPressedCleanupTime)
   }
+
+  private getToggleIndicatorIconName = (): string => {
+    const { open } = this.state
+    const positionedAbove = this.props.position === 'above'
+
+    return (open && !positionedAbove) || (!open && positionedAbove)
+      ? 'stardust-arrow-up'
+      : 'stardust-arrow-down'
+  }
```

Comment:
This behavior is very strange, native `select` doesn't do the same thing:

![image](https://user-images.githubusercontent.com/14183168/58541553-0c52b780-81fc-11e9-9a3c-4375c13c9490.png)

Reply:
removed logic for now, will create issue

Reply:
One of the aria examples is broken